### PR TITLE
[DFFS-4] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by Seyi01**
Screenshot (9)
Screenshot (10)

### CODE REVIEW

**Review by Ireti203 — CHANGES_REQUESTED** [review]
The index.html file should be in the HTML folder.

**Comment by Seyi01** [comment]
Done ma. Thank you.
Screenshot (11)

**Comment by Seyi01** [comment]
Done ma. thank you
Screenshot (11)

File: submissions/seyifunmiOjo/html-forms/contact.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/seyifunmiOjo/html-forms/contact.html:1]: You already created `contact page` for your previous task, why creating it again?

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by Seyi01** [comment]
Hi @Ireti203. I have understood and done the needful. There is only one contact page in the submissions folder and the nav links have been added to both it and the products page. Pls review. Thank you.

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**